VERSION.txt

```diff
@@ -1 +1 @@
-10.3.0
+11.0.0
```

Comment:
I'd argue that this is not a breaking change and is still new functionality i.e it should become 10.4.0 (https://semver.org/)

Reply:
fair point, il get that updated